src/app/api/auth/forgot-password/route.ts

```diff
@@ -17,7 +19,15 @@ export default async function POST(request: NextRequest) {
     });
     if (user !== null) {
       const token = sign({ email }, process.env.NEXTAUTH_SECRET ?? "");
+      void mailer.sendMail({
```

Comment:
In theory, the behavior should be correct, but the problem seems to be on Vercel. The function terminates early if the expression is not awaited, so there's no way to avoid this.